### PR TITLE
API-11729: Add benefit types to contestable issues endpoint docs

### PR DESCRIPTION
**Pull request by ElleeB**
Refactors rswag for contestable issues endpoint so that the acceptable benefit types appear in the documentation.
Also snuck in a little update for the decision review type to accomplish the same.

Ticket: https://vajira.max.gov/browse/API-11729

### CODE REVIEW

File: modules/appeals_api/app/swagger/appeals_api/v2/swagger.json [hunk 1]
```diff
@@ -3095,18 +3095,7 @@
           {
             "name": "benefit_type",
             "in": "query",
-            "description": "Benefit Type for the appeal. Required if Decision Review is a Higher Level Review.",
-            "enum": [
```
Comment (cilestin) [inline modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3099]: Why were these enums removed? Did they not end up showing in the documentation?

Comment (ElleeB) [reply modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3099]: nope. enum is not a thing for the swagger documentation.

Comment (ElleeB) [reply modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3099]: doesn't do anything

Comment (ElleeB) [reply modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3099]: in this context the benefit type is a parameter...not schema.

Comment (ElleeB) [reply modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3099]: Note: Drew and I discussed possibly adding the enum as schema and then referencing the schema in the parameters documentation.

**Review by thilton-oddball — APPROVED** [review]

File: modules/appeals_api/app/swagger/appeals_api/v2/swagger.json [hunk 2]
```diff
@@ -3089,26 +3089,31 @@
             "required": true,
             "description": "Scoping of appeal type for associated issues",
             "schema": {
-              "type": "string"
+              "type": "string",
+              "enum": [
+                "higher_level_reviews",
+                "notice_of_disagreements",
+                "supplemental_claims"
```
Comment (cilestin) [inline modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3096]: We shouldn't add SC's to prod documentation just yet.

File: modules/appeals_api/app/swagger/appeals_api/v2/swagger.json [hunk 3]
```diff
@@ -3089,26 +3089,31 @@
...
-              "pensionSurvivorsBenefits",
-              "fiduciary",
-              "lifeInsurance",
-              "veteransHealthAdministration",
-              "veteranReadinessAndEmployment",
-              "loanGuaranty",
-              "education",
-              "nationalCemeteryAdministration"
-            ],
+            "description": "Required if decision review type is Higher Level Review or Supplemental Claim.",
```
Comment (cilestin) [inline modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3103]: Let's not mention SC's here on prod documentation until we've done E2E testing in the lower envs & released the endpoint to prod.

**Review by cilestin — CHANGES_REQUESTED** [review]

File: modules/appeals_api/app/swagger/appeals_api/v2/swagger.json (hunk 3 above)
Comment (ElleeB) [reply modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3103]: oh right!

File: modules/appeals_api/app/swagger/appeals_api/v2/swagger.json (hunk 2 above)
Comment (ElleeB) [reply modules/appeals_api/app/swagger/appeals_api/v2/swagger.json:3096]: RIGHT!

**Review by cilestin — APPROVED** [review]

**Pull request merged**